compiler/record-minmax/driver/Driver.cpp

```diff
@@ -25,7 +26,13 @@ int entry(const int argc, char **argv)
   auto input_data_path = args.getInputDataFilePath();
   auto output_model_path = args.getOutputModelFilePath();
 
-  // TODO: Implement record_minmax functionality
+  RecordMinMax rmm(input_model_path);
```

Comment:
(suggestion) it would be better not to do something in CTOR that may fail, throw, exit.
I would use something like `open()` and get the result or provide caller to use `try-catch`

Reply:
I've made a new function named 'initialize' and do initialization in that function. Thanks!